src/link/edit.js

```diff
@@ -193,18 +207,33 @@ const Edit = ( {
 		}
 	}, [ value ] );
 
+	useEffect( () => {
+		if ( customTooltipDisplayedCount < customTooltipDisplayedLimit ) {
+			// Wait 1 second and then display tooltip
+			waitOneSecThenDisplayCustomTooltip();
+		}
+	} );
+
 	return (
 		<>
 			<BlockControls>
 				<ToolbarGroup>
 					<ToolbarButton
 						icon={ generateWikipediaLogo( isActive ? 'white' : 'black' ) }
-						title={ formatTitle }
+						title={ __( 'Add Wikipedia Preview', 'wikipedia-preview' ) }
 						isActive={ isActive }
 						onClick={ formatButtonClick }
+						className="wikipediapreview-edit-toolbar-button"
```

Comment:
do we need this className here?

Reply:
You're right, that's left over from original approach